core/common/src/Buffer.kt

```diff
@@ -39,17 +39,26 @@ import kotlin.jvm.JvmField
  * does not affect buffer's state and [exhausted] only indicates that a buffer is empty.
  */
 public class Buffer : Source, Sink {
-    @JvmField
+    @PublishedApi
+    @get:JvmName("#int#getHead")
```

Comment:
just curious:
1. why `JvmName` is needed here (and in all other places)
2. why it has such name pattern?

Reply:
To prevent use of these functions/properties from Java. Names were chosen with that in mind, `int` stands for `internal`.

All recently annotated API was, of course, directly accessible from Java before.
But as `Segment` is now public, it might feels like an invitation to use its API.

Reply:
@fzhinkin just curious, why not use `@JvmSynthetic`?

Reply:
@lppedd No particular reason. Switched to `JvmSynthetic` as it seems to be a less controversial option :)